csharp/test/Drivers/Databricks/DatabricksConnectionTest.cs

```diff
@@ -315,5 +318,31 @@ public InvalidConnectionParametersTestData()
                 Add(new(new() { [SparkParameters.Type] = SparkServerTypeConstants.Http, [SparkParameters.HostName] = "valid.server.com", [AdbcOptions.Username] = "user", [AdbcOptions.Password] = "myPassword", [DatabricksParameters.TemporarilyUnavailableRetryTimeout] = "-1" }, typeof(ArgumentOutOfRangeException)));
             }
         }
+
+        /// <summary>
+        /// Tests that default namespace is correctly stored in the connection.
+        /// </summary>
+        [SkippableFact]
+        internal void DefaultNamespaceStoredInConnection()
+        {
+            // Skip if default catalog or schema is not configured
+            Skip.If(string.IsNullOrEmpty(TestConfiguration.DefaultCatalog), "Default catalog not configured");
+            Skip.If(string.IsNullOrEmpty(TestConfiguration.DefaultSchema), "Default schema not configured");
+
+            // Act
+            using var connection = NewConnection();
+
+            // Assert
+            Assert.NotNull(connection);
+            Assert.IsType<DatabricksConnection>(connection);
+
+            var defaultNamespaceProperty = typeof(DatabricksConnection).GetProperty("DefaultNamespace", BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.Public);
```

Comment:
With `InternalsVisibleToAttribute` and an internal property, it shouldn't be necessary to use Reflection to get at this property value.